hack/components/components.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+
+set -xe
+
+source hack/components/common_functions.sh
+CNAO_VERSION=${VERSION} # Exported from Makefile
+
+echo 'Setup temporary Go path'
+export GOPATH=${PWD}/_components/go
+mkdir -p $GOPATH
+export PATH=${GOPATH}/bin:${PATH}
+
+echo 'kubemacpool'
+source hack/components/bump_kubemacpool.sh
+
+echo 'macvtap'
+source hack/components/bump_macvtap.sh
+
+echo 'kubernetes-nmstate'
+source hack/components/bump_knmstate.sh
```

Comment:
Let's do a loop for this and has a var about components `components="kubemacpool macvtap kubernetes-nmstate` we will need to rename `bump_knmstate.sh` to `bump_kubernetes-nmtate`

Reply:
np I'll add it in the next PR #449